rivershared/riverpilot/pilot.go

```diff
@@ -26,7 +26,7 @@ type Pilot interface {
 	JobInsertMany(
 		ctx context.Context,
 		tx riverdriver.ExecutorTx,
-		params []*riverdriver.JobInsertFastParams,
+		params *riverdriver.JobInsertFastManyParams,
```

Comment:
I'm sure you've already considered it but this will definitely require corresponding changes in Pro. Then again so will a lot of this PR :)

Reply:
Yeah, I think I finished most of those up this evening thankfully.  I'll send them over today or tomorrow.